src/components/modal/nz-modal.service.ts

```diff
@@ -81,16 +82,20 @@ export class NzModalService {
       }
     });
 
-    props[ 'onOk' ] = this._getConfirmCb(props[ 'nzOnOk' ]);
+    const isShowConfrimLoading = !!config[ 'showConfirmLoading' ];
+    props[ 'onOk' ] = this._getConfirmCb(props[ 'nzOnOk' ], isShowConfrimLoading);
     props[ 'onCancel' ] = this._getConfirmCb(props[ 'nzOnCancel' ]);
     // 在service模式下，不需要nzOnOk，防止触发this.nzOnOk.emit(e);
     delete props[ 'nzOnOk' ];
     delete props[ 'nzOnCancel' ];
     return props;
   }
 
-  _getConfirmCb(fn?: Function): Function {
-    return (_close) => {
+  _getConfirmCb(fn?: Function, isShowConfrimLoading: boolean = false): Function {
```

Comment:
Hi, @AlcheXie thanks for your pr.
It would be better submit an issue before pr any new feat, and doc needs updated when new feat added.

Reply:
Ok, I see.